app/tests/api/activity_value.test.ts

```diff
@@ -478,6 +486,5 @@ describe("Activity Value API", () => {
 
     const { data } = await res3.json();
     assert.equal(res3.status, 200);
-    assert.equal(data.deletedCount, 3);
```

Comment:
Why did you need to remove this assertion? It should still be correct, right?

Reply:
This tests has to be re-modelled, I plan to rewrite them as they are flaky